packages/cli/src/lib/flags.ts

```diff
@@ -67,6 +67,12 @@ export const commonFlags = {
       char: 'e',
     }),
   },
+  envHandle: {
```

Comment:
What do you think about calling this `environment` or `env`?

Reply:
Agreed since those are unused now.